src/pages/installation.md

```diff
@@ -0,0 +1,181 @@
+- **Portfolio** - Astro's Portfolio Starter Template. Here  you can experiment and build professional portfolio sites with Astro
+<!-- I think we should have guides for each of these templates, perhaps in a template section, with tutorials on building something with each of them, -->
+
+We would recommend to start with the 'Starter Kit' template. This would give us a solid platform to liftoff from.
+
+Astro will begin to create and setup the project files for you, once done you will be prompted with the post-installation next steps.
+
+```bash
+  1: npm install (or pnpm install, yarn, etc)
+  2: git init && git add -A && git commit -m "Initial commit" (optional step)
+  3: npm start (or pnpm, yarn, etc)
+```
+
+At this stage, Astro has merely been copied to your new project. We would need to install the Astro dependencies and setup our `node_modules` folder inside the project.
+
+```bash
+npm install || yarn
+```
+
+This process will first, fetch the required dependencies from the npm registry, and then link and build the packages out for Astro to be able to use.
+
+```bash
+git init && git add -A && git commit -m "Initial commit"
+```
+
+This step, is optional, we do recommend setting up and adding `git` to the project. This combined command will let you set this up with ease.
+
+Astro adds only two commands to the `scripts` in the `package.json` file:
+
+```js
+{
+  "scripts": {
+    "start": "astro dev",
+    "build": "astro build"
+  }
+}
+
+```
+To develop your site in Astro, run the following command:
+
+```bash
+npm run start || yarn start || pnpm start
+```
+
+It starts the Astro **dev** server. This would start [Snowpack](https://snowpack.dev), which prepares all the Astro dependencies. This is a 'one-time' step and the results of which would be cached for the lifetime of the project.
+
+Once Snowpack has completed its task, you would be able to view the `dev` server at the default address of: `http://127.0.0.1:3000/`
+
+<!-- This would be a great place to tie in an Explore Astro Section or link to -->
+
+To build your Astro project ready for deployment, enter:
+
+```bash
+npm run build || yarn build || pnpm build
+```
+
+Astro would now build your site and outputs the compiled HTML and assets to the newly created `./dist` directory in your project root. This can then be deployed to any of your favourite hosting provider.
+
+## Manual Installation
+
+If you choose to setup Astro, without the use of Astro's CLI Tool, you can do so. A lot of the setup is similar to the process outlined above, with subtle differences. In this part we will walk you through how to get setup with Astro using good old elbow grease.
+
+- First lets create the directory for your project:
+
+```bash
+mkdir my-Astro-project
+```
+
+- Navigate into the folder:
+
+```bash
+cd my-Astro-project
+```
+
+- Create a `package.json` file:
+
+```bash
+npm init (-y?) || yarn init 
```

Comment:
I believe we can give the user some options, one using the npm init with the link that explains it (https://docs.npmjs.com/cli/v7/commands/npm-init) or just ask to create a file named package.json and add `{ }` to it maybe also with some link that explains more about package.json files

Reply:
Do you mean; link the commands to the relevant pkg mgnrs init pages, I think that would be a good idea, just wanting to clarify

Reply:
Oh sorry, didn't see I put something just for npm, but yes, a link with the info about how to do it with their package manager, but also the possibility to create the file themselves without the use of package managers.